### ISSUE TITLE
Incompatability with rsync 3.2.4 appears to be fixed but stated as still a problem in readme

### ISSUE THREAD

**Issue by ianwillie** [bit-team/backintime#1272]
Before I begin, thank you team for taking over the development of BackInTime.

I see this comment in readme for backintime (BiT) 1.3.2, March 2022: (https://github.com/bit-team/backintime)
    "Back In Time is currently incompatible with rsync >= 3.2.4, April 2022.
    If you use rsync >= 3.2.4, as a workaround,
        add "--old-args" in "Expert Options / Additional options to rsync" and
        Modify /usr/bin/backintime
            to include export RSYNC_OLD_ARGS=1."

First:  the rsync changelog for v 3.2.4 says: (https://download.samba.org/pub/rsync/NEWS#3.2.5)
    "If your rsync script depends on the old arg-splitting behavior, **either** run it with the [--old-args](https://download.samba.org/pub/rsync/rsync.1#opt--old-args) option **or** export RSYNC_OLD_ARGS=1 in the script's environment."
Note that it is either one fix or the other but not both fixes as stated in the BiT readme quoted above.

Two: In my version of BiT 1.3.2-2 obtained from Manjaro official repository 20 August 2022 "export RSYNC_OLD_ARGS=1" exists in  /usr/bin/backintime:
 ```
CUR_PATH="$(dirname $(readlink -m $0))"
        if [ -f "${CUR_PATH}/backintime.py" ]; then
	        APP_PATH=$CUR_PATH
        else
	        APP_PATH=$(readlink -m "${CUR_PATH}/../share/backintime/common")
        fi
        
        export RSYNC_OLD_ARGS=1
        python -Es $APP_PATH/backintime.py "$@"

```

As a result it appears that that BiT 1.3.2 -2 the incompatibility with rsync has been fully handled because the additional option of "--old-args" is not required, the export command being sufficient. If this is the case, perhaps one of the development team would confirm this and I suggest that they change the readme to reflect that BiT has fully corrected this. (Well done!)

I am just so glad that the current developers have taken on to maintain this wonderful program that many of us depend upon. I have used it for many years and on frequent occasions it has averted disaster because it is so easy to reinstall files from the dated backups.  I have a data partition of about 300Gb and I often use the BiT as a simple alternative to using version control in my development python and R scripts.
Kind wishes.

**Comment by Codeberg-AsGithubAlternative-buhtz** [comment]
Thanks for your report. I am not sure what is going on but it seems to me that Manjaro itself patched that befor whe at upstream could do it.

We will take that into account when patching.

**Comment by ianwillie** [comment]
Thanks for quick reply. Thanks for pointing out the other related posts.
I guess that having "export RSYNC_OLD_ARGS=1"  in /usr/bin/backintime will be fine with earlier versions of rsync before 3.2.4. However, I imagine that having "--old-args" in earlier versions of the rsync call might cause problems with the inclusion of an unrecognised option. Hence it is probably better not to include BOTH commands. I do not have an older version of rsync installed to check this and I am too new to Manjaro to fiddle with downgrading till I have more experience.

**Comment by aryoda** [comment]
Related to #1247 (see my proposed patch there). Would be great if some could test the patched version with the new `rsync` version 3.2.4++
